pkg/accounts/adaptor/controller/account.ts

```diff
@@ -212,24 +221,38 @@ export class AccountController {
     if (Result.isErr(res)) {
       return res;
     }
+    const account = Result.unwrap(res);
+
+    const avatarRes = await this.avatarService.fetchByAccountID(
+      account.getID(),
+    );
+    const headerRes = await this.headerService.fetchByAccountID(
+      account.getID(),
+    );
+    const avatar = Result.isOk(avatarRes)
+      ? Result.unwrap(avatarRes).getUrl()
+      : '';
```

Comment:
```suggestion
    const avatar = Result.mapOr('')((avatar) => avatar.getUrl())(avatarRes);
```
[`mapOr`](https://mikuroxina.github.io/mini-fn/functions/Result.mapOr.html) を使ったほうがよいかと思います. 他の三項演算子と `unwrap` で書いているところも同様です.